active_manifesto_experiments.py

```diff
@@ -184,23 +187,33 @@ def run_experiment(
         'percentage_samples':percentage_samples
         }
 
-    json.dump(results,open("active_learning_curves.json","wt"))
+    json.dump(results,open(output_filename,"wt"))
     return results
 
 
-def plot_results(fn):
+def plot_results(fn=EXPERIMENT_RESULT_FILENAME):
     import pylab
     results = json.load(open(fn))
     ac = sp.median(sp.vstack(results['active_learning_curves']), axis=0)
+    ac_sd = np.std(sp.vstack(results['active_learning_curves']), axis=0)
```

Comment:
std is glaub gut wenn der mean geplotted wird, fuer median isses besser, percentiles zu nehmen, 25/75 oder 05/95 oder so.

Reply:
wollte eh noch alle sampling strategies im plot mal miteinander vergleichen, dazu wollte ich noch die struktur des result files aendern, so als dataframe, mit ner column fuer nen indicator fuer die sampling strategy, und dann kann man sns tsplot http://seaborn.pydata.org/generated/seaborn.tsplot.html benutzen ...